pandas/io/parsers/python_parser.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
could do this and pandas/io/parsers/base_parser.py in a pre-cursor to reduce the noise here.

Reply:
will do